cli/command/trust/signer_remove.go

```diff
@@ -0,0 +1,137 @@
+package trust
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/trust"
+	"github.com/docker/docker/api/types"
+	registrytypes "github.com/docker/docker/api/types/registry"
+	"github.com/docker/notary/client"
+	"github.com/docker/notary/tuf/data"
+	"github.com/spf13/cobra"
+)
+
+type signerRemoveOptions struct {
+	forceYes bool
+}
+
+func newSignerRemoveCommand(dockerCli command.Cli) *cobra.Command {
+	options := signerRemoveOptions{}
+	cmd := &cobra.Command{
+		Use:   "remove [OPTIONS] NAME IMAGE [IMAGE...]",
+		Short: "Remove a signer",
+		Args:  cli.RequiresMinArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return removeSigner(dockerCli, args[0], args[1:], &options)
```

Comment:
Most commands (including `newSignerAddCommand` above) pass args using the options struct instead of separate args. I think it would be good to do that here as well.

Also it shouldn't be necessary to use a pointer for options. The values should never be modified, so I believe we pass a struct (not a pointer) in all other commands.